src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -1008,6 +1025,12 @@ public static void UnregisterGame(Game game)
 			}
 		}
 
+		private unsafe static int MeasureStringLength(byte* ptr)
+		{
+			int bytes;
+			for (bytes = 0; *ptr != 0; ptr += 1, bytes += 1);
+			return bytes;
+		}
```

Comment:
Agh dangit, this one's my fault.
```suggestion
		}

```